pkg/apis/kops/cluster.go

```diff
@@ -249,7 +249,9 @@ type ScalewaySpec struct {
 }
 
 type KarpenterConfig struct {
-	Enabled bool `json:"enabled,omitempty"`
+	Enabled   bool   `json:"enabled,omitempty"`
+	Version   string `json:"version,omitempty"`
+	LogFormat string `json:"logFormat,omitempty"`
```

Comment:
Let's keep this in alphabetical order.
```suggestion
	LogFormat string `json:"logFormat,omitempty"`
	Version   string `json:"version,omitempty"`
```

Reply:
Fixed :slightly_smiling_face: